cpp/tools/parquet/CMakeLists.txt

```
@@ -31,6 +31,7 @@ if(PARQUET_BUILD_EXECUTABLES)
     install(TARGETS ${TOOL} ${INSTALL_IS_OPTIONAL}
             RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
   endforeach(TOOL)
+  target_link_libraries(parquet-dump-footer thrift)
```

Could you use `thrift::thrift` as the CMake target name for Apache Thrift?

```suggestion
  target_link_libraries(parquet-dump-footer thrift::thrift)
```

---

Done.